python_api/my_python_api/models.py

```diff
@@ -1,13 +1,41 @@
 from .database import Base
-from sqlalchemy import Column, Integer, String, Boolean
-from sqlalchemy.sql.sqltypes import TIMESTAMP
-from sqlalchemy.sql.expression import text
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, Date, Enum
+from sqlalchemy.orm import relationship
+import enum
 
+class Status(enum.Enum):
+    PROPOSED = 0
+    IN_PROGRESS = 1
+    COMPLETED = 2
+    
 class User(Base):
     __tablename__ = "users"
     
-    firstName = Column(String, nullable=False)
-    lastName = Column(String, nullable=False)
-    email = Column(String, primary_key=True, nullable=False) 
-    password = Column(String, nullable=True)
+    email = Column(String(length=255), nullable=False, unique=True)
+    user_id = Column(String(length=36), primary_key=True, nullable=False)
+    first_name = Column(String(length=50), nullable=True)
+    last_name = Column(String(length=50), nullable=True)
+    
+class Project(Base):
+    __tablename__ = "projects"
+    
+    tasks = relationship("Task", backref="project")
```

Comment:
Im not to sure for the syntax. Does this allow for multiple Tasks to be assigned? as of right now it seems that only 1 task will be related to a project instead of multiple. Then again im not too sure how this syntax works